xdsl/interpreters/experimental/pdl.py

```diff
@@ -284,6 +284,14 @@ def run_result(
         assert isinstance(parent, Operation)
         return (parent.results[op.index.value.data],)
 
+    @impl(pdl.AttributeOp)
+    def run_attribute(
+        self, interpreter: Interpreter, op: pdl.AttributeOp, args: tuple[Any, ...]
+    ) -> tuple[Any, ...]:
+        op.verify()
```

Comment:
why the verify?

Reply:
Not really sure, just thought it would be useful. I'll take it out